nuget/helpers/lib/NuGetUpdater/.editorconfig

```diff
@@ -20,6 +20,7 @@ tab_width = 4
 
 # New line preferences
 insert_final_newline = true
+#end_of_line = lf
```

Comment:
I have my local environment set to always check out with `LF` so adding this makes it easy for me to `dotnet format`.